packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -170,15 +170,19 @@ export class SingleAddressWallet implements ObservableWallet {
       connectionStatusTracker$ = createSimpleConnectionStatusTracker()
     }: SingleAddressWalletDependencies
   ) {
-    this.#logger = logger;
+    this.#logger = contextLogger(logger, 'SingleAddressWallet');
     this.#inputSelector = inputSelector;
+
+    this.#logger.debug('Creating providers needed for syncStatus tracking');
```

Comment:
Constructor is all synchronous, if something fails then `new SingleAddressWallet` would throw. I would minimize those logs to just 'initializing' and 'initialized'

Reply:
Done in [3eeba2a](https://github.com/input-output-hk/cardano-js-sdk/pull/379/commits/3eeba2a461134843d202116d39c1edec2c118116)

Reply:
Maybe also add an 'Initialized' message, because right now just reading the logs it's not clear that it ever initialized given the 'Initializing...' message